modules/setting/storage.go

```diff
@@ -220,5 +222,10 @@ func getStorage(rootCfg ConfigProvider, name, typ string, sec ConfigSection) (*S
 		}
 	}
 
+	if storageTarget, ok := storagePaths[storage.Path]; ok && storageTarget != name {
```

Comment:
It's not right for minio configuration. Just check storage.Path for local type.

Reply:
Done.

Reply:
Every path should be unique.

Because `DeleteRepositoryArchives` calls `storage.Clean(storage.RepoArchives)`, it deletes all files in a "storage".

This function's dangerous behavior should also be fixed.